### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by ojarmes** [open-as-team/open-as-cgw#45]
Ubuntu 14.04.05 LTS
Latest Updates

Install:
sudo bash -c "$(curl -sL openas.org/install-devel.sh)"

console output:
login as: adminlocal
adminlocal@192.168.123.38's password:
Welcome to Ubuntu 14.04.5 LTS (GNU/Linux 4.4.0-36-generic x86_64)

 * Documentation:  https://help.ubuntu.com/

  System information as of Sun Sep  4 18:59:05 CEST 2016

  System load:  0.0                Processes:           135
  Usage of /:   1.4% of 116.77GB   Users logged in:     0
  Memory usage: 11%                IP address for eth0: 192.168.123.38
  Swap usage:   0%

  Graph this data and manage this system at:
    https://landscape.canonical.com/

New release '16.04.1 LTS' available.
Run 'do-release-upgrade' to upgrade to it.

Your Hardware Enablement Stack (HWE) is supported until April 2019.
Last login: Sun Sep  4 18:59:04 2016
adminlocal@open-as:~$ sudo bash -c "$(curl -sL openas.org/install-devel.sh)"
[sudo] password for adminlocal:
  ____                  ___   ____
 / __ \___  ___ ___    / _ | / __/
/ /_/ / _ \/ -_) _ \  / __ |_\ \
\____/ .__/\__/_//_/ /_/ |_/___/
    /_/

[ Open AS Communication Gateway - www.openas.org ]
[ Install Script for Ubuntu LTS 14.04 Trusty ]
[ (c) 2016 by the Open AS Team ]

This script will install the latest Open AS Communication Gateway.
Please make sure you have a backup before starting the installation.

! Development release - do not use in production environments !

Are you sure you want to continue? <y/N> y

*** Detecting OS version... OK
*** Setting debconf variables... OK
*** Starting installation via ppa... OK
gpg: keyring `/tmp/tmpppxtbu8y/secring.gpg' created
gpg: keyring `/tmp/tmpppxtbu8y/pubring.gpg' created
gpg: requesting key AE65E7F7 from hkp server keyserver.ubuntu.com
gpg: /tmp/tmpppxtbu8y/trustdb.gpg: trustdb created
gpg: key AE65E7F7: public key "Launchpad PPA for Open AS Team" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
OK
Hit http://security.ubuntu.com trusty-security InRelease
Get:1 http://ppa.launchpad.net trusty InRelease [15.4 kB]
Hit http://security.ubuntu.com trusty-security/main Sources
Get:2 http://ppa.launchpad.net trusty/main amd64 Packages [14 B]
Hit http://security.ubuntu.com trusty-security/restricted Sources
Hit http://security.ubuntu.com trusty-security/universe Sources
Hit http://security.ubuntu.com trusty-security/multiverse Sources
Ign http://us.archive.ubuntu.com trusty InRelease
Hit http://security.ubuntu.com trusty-security/main amd64 Packages
Hit http://security.ubuntu.com trusty-security/restricted amd64 Packages
Hit http://security.ubuntu.com trusty-security/universe amd64 Packages
Get:3 http://ppa.launchpad.net trusty/main i386 Packages [14 B]
Hit http://security.ubuntu.com trusty-security/multiverse amd64 Packages
Hit http://security.ubuntu.com trusty-security/main i386 Packages
Get:4 http://ppa.launchpad.net trusty/main Translation-en [14 B]
Get:5 http://us.archive.ubuntu.com trusty-updates InRelease [65.9 kB]
Hit http://security.ubuntu.com trusty-security/restricted i386 Packages
Hit http://security.ubuntu.com trusty-security/universe i386 Packages
Hit http://security.ubuntu.com trusty-security/multiverse i386 Packages
Hit http://security.ubuntu.com trusty-security/main Translation-en
Hit http://security.ubuntu.com trusty-security/multiverse Translation-en
Hit http://security.ubuntu.com trusty-security/restricted Translation-en
Hit http://security.ubuntu.com trusty-security/universe Translation-en
Hit http://us.archive.ubuntu.com trusty-backports InRelease
Hit http://us.archive.ubuntu.com trusty Release.gpg
Get:6 http://us.archive.ubuntu.com trusty-updates/main Sources [380 kB]
Get:7 http://us.archive.ubuntu.com trusty-updates/restricted Sources [5,360 B]
Get:8 http://us.archive.ubuntu.com trusty-updates/universe Sources [163 kB]
Get:9 http://us.archive.ubuntu.com trusty-updates/multiverse Sources [7,137 B]
Get:10 http://us.archive.ubuntu.com trusty-updates/main amd64 Packages [889 kB]
Get:11 http://us.archive.ubuntu.com trusty-updates/restricted amd64 Packages [15.9 kB]
Get:12 http://us.archive.ubuntu.com trusty-updates/universe amd64 Packages [373 kB]
Get:13 http://us.archive.ubuntu.com trusty-updates/multiverse amd64 Packages [14.8 kB]
Get:14 http://us.archive.ubuntu.com trusty-updates/main i386 Packages [851 kB]
Get:15 http://us.archive.ubuntu.com trusty-updates/restricted i386 Packages [15.6 kB]
Get:16 http://us.archive.ubuntu.com trusty-updates/universe i386 Packages [374 kB]
Get:17 http://us.archive.ubuntu.com trusty-updates/multiverse i386 Packages [15.2 kB]
Hit http://us.archive.ubuntu.com trusty-updates/main Translation-en
Hit http://us.archive.ubuntu.com trusty-updates/multiverse Translation-en
Hit http://us.archive.ubuntu.com trusty-updates/restricted Translation-en
Hit http://us.archive.ubuntu.com trusty-updates/universe Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/main Sources
Hit http://us.archive.ubuntu.com trusty-backports/restricted Sources
Hit http://us.archive.ubuntu.com trusty-backports/universe Sources
Hit http://us.archive.ubuntu.com trusty-backports/multiverse Sources
Hit http://us.archive.ubuntu.com trusty-backports/main amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/restricted amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/universe amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/multiverse amd64 Packages
Hit http://us.archive.ubuntu.com trusty-backports/main i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/restricted i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/universe i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/multiverse i386 Packages
Hit http://us.archive.ubuntu.com trusty-backports/main Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/multiverse Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/restricted Translation-en
Hit http://us.archive.ubuntu.com trusty-backports/universe Translation-en
Hit http://us.archive.ubuntu.com trusty Release
Hit http://us.archive.ubuntu.com trusty/main Sources
Hit http://us.archive.ubuntu.com trusty/restricted Sources
Hit http://us.archive.ubuntu.com trusty/universe Sources
Hit http://us.archive.ubuntu.com trusty/multiverse Sources
Hit http://us.archive.ubuntu.com trusty/main amd64 Packages
Hit http://us.archive.ubuntu.com trusty/restricted amd64 Packages
Hit http://us.archive.ubuntu.com trusty/universe amd64 Packages
Hit http://us.archive.ubuntu.com trusty/multiverse amd64 Packages
Hit http://us.archive.ubuntu.com trusty/main i386 Packages
Hit http://us.archive.ubuntu.com trusty/restricted i386 Packages
Hit http://us.archive.ubuntu.com trusty/universe i386 Packages
Hit http://us.archive.ubuntu.com trusty/multiverse i386 Packages
Hit http://us.archive.ubuntu.com trusty/main Translation-en
Hit http://us.archive.ubuntu.com trusty/multiverse Translation-en
Hit http://us.archive.ubuntu.com trusty/restricted Translation-en
Hit http://us.archive.ubuntu.com trusty/universe Translation-en
Ign http://us.archive.ubuntu.com trusty/main Translation-en_US
Ign http://us.archive.ubuntu.com trusty/multiverse Translation-en_US
Ign http://us.archive.ubuntu.com trusty/restricted Translation-en_US
Ign http://us.archive.ubuntu.com trusty/universe Translation-en_US
Fetched 3,186 kB in 9s (337 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package open-as-cgw
dpkg-query: no packages found matching open-as-cgw
-----------------------------------------------------------
OPEN AS Communication Gateway installation FAILED!
-----------------------------------------------------------
Please copy the console output and the dpkg and apt logs
located at /var/log and open a new ticket in our bugtracker
Thank you in advance!
-----------------------------------------------------------
adminlocal@open-as:~$

**Comment by krisgermann** [comment]
I have also run into this error even after manually adding the repository and PPA.

**Comment by allegiancetech** [comment]
Me too on clean 14.04.05 LTS.  Any resolution?  Any news on the new version "coming soon"?

**Comment by krisgermann** [comment]
sudo vi /etc/apt/sources.list
sudo apt-get update
sudo bash -c "$(curl -sL openas.org/install-devel.sh)"

If that fails, compile from package @ https://launchpad.net/~open-as-team/+archive/ubuntu/ppa/+files/open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb

sudo wget https://launchpad.net/~open-as-team/+archive/ubuntu/ppa/+files/open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb
sudo dpkg -i open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb

**Comment by allegiancetech** [comment]
Thanks for the quick response.  But, what do I add to the /etc/apt/sources.list file?

**Comment by krisgermann** [comment]
I edited my reply, but this:

deb http://ppa.launchpad.net/open-as-team/ppa/ubuntu xenial main
deb-src http://ppa.launchpad.net/open-as-team/ppa/ubuntu xenial main

**Comment by allegiancetech** [comment]
Thanks for the help.  It still failed, so I will try compiling it on 16.04

**Comment by krisgermann** [comment]
No problem - what's the failure error?

**Comment by allegiancetech** [comment]
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 open-as-cgw : PreDepends: libauthen-htpasswd-perl but it is not installable
E: Unable to correct problems, you have held broken packages.
dpkg-query: no packages found matching open-as-cgw
-----------------------------------------------------------
OPEN AS Communication Gateway installation FAILED!
-----------------------------------------------------------
Please copy the console output and the dpkg and apt logs
located at /var/log and open a new ticket in our bugtracker
Thank you in advance!
-----------------------------------------------------------

**Comment by krisgermann** [comment]
As the first (easiest) way has failed, please compile from the package using dpkg per instruction up here https://github.com/open-as-team/open-as-cgw/issues/45#issuecomment-255842307

**Comment by schallertd** [comment]
It's broken atm., dont bother trying.

**Comment by krisgermann** [comment]
Thanks for the tip :)

**Comment by ojarmes** [comment]
____                  ___   ____
 / __ \___  ___ ___    / _ | / __/
/ /_/ / _ \/ -_) _ \  / __ |_\ \  
\____/ .__/\__/_//_/ /_/ |_/___/  
    /_/                           

[ Open AS Communication Gateway - www.openas.org ]
[ Install Script for Ubuntu LTS 14.04 Trusty ]
[ (c) 2016 by the Open AS Team ]

This script will install the latest Open AS Communication Gateway.
Please make sure you have a backup before starting the installation.

! Development release - do not use in production environments !

Are you sure you want to continue? <y/N> y

*** Detecting OS version... [ERROR] OS not supported! 
root@openas:/tmp# 

root@openas:/tmp# sudo dpkg -i open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb
dpkg: regarding open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb containing open-as-cgw, pre-dependency problem:
 open-as-cgw pre-depends on amavisd-new
  amavisd-new is not installed.

dpkg: error processing archive open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb (--install):
 pre-dependency problem - not installing open-as-cgw
Errors were encountered while processing:
 open-as-cgw_2.2.0-0~221~ubuntu16.04.1_all.deb
root@openas:/tmp#

**Comment by krisgermann** [comment]
Per @schallertd in https://github.com/open-as-team/open-as-cgw/issues/45#issuecomment-255863559

"It's broken atm., sorry."

For debugging purposes, please read through the console output:

"*** Detecting OS version... [ERROR] OS not supported!"

Are you running 14.04? If you are, try with 14.04.5 (Clean installs only).
